orchagent/qosorch.cpp

```diff
@@ -616,15 +617,15 @@ sai_object_id_t QosOrch::initSystemAclTable()
     sai_status_t status;
 
     /* Create system ACL table */
-    attr.id = SAI_ACL_TABLE_ATTR_BIND_POINT;
-    attr.value.s32 = SAI_ACL_BIND_POINT_SWITCH;
+    attr.id = SAI_ACL_TABLE_ATTR_ACL_BIND_POINT_TYPE_LIST;
+    attr.value.s32 = SAI_ACL_BIND_POINT_TYPE_SWITCH;
```

Comment:
Do we need to consider LAG here? also @lguohan

---
In reply to: [111073589](https://github.com/lguohan/sonic-swss/pull/1#discussion_r111073589) [](ancestors = 111073589)